chapter02/rwd/ex5-2.pl

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/perl
+
+print "Give me a string: ";
+my $s = <>;
+
+print "Give me a number: ";
+my $n = <>;
+
+print "The product is: " . $s x $n ."\n";
```

Comment:
I have tried this code, and it actually works. I think it's because chomp method is not used, so the "new line" character is picked up when we input the string values with Enter Key.